docs/user-guide.md

```diff
@@ -145,6 +145,10 @@ To import JSON or a zip file with conversations, click the **Import conversation
  
 ![](./img/import_conv.png)
 
+> Note, that when importing a duplicate of an existing conversation, you will be prompted to select one of the proceeding options: replace, ignore or add a prefix to the imported conversation.
```

Comment:
How do you think? does it make sense to note here that attached files are also will be asked to replace, ignor, prefix?